ui/webui/test/anacondalib.py

```diff
@@ -43,6 +43,12 @@ def setUp(self):
         self.machine.execute("systemctl restart cockpit")
         self.allow_journal_messages('.*cockpit.bridge-WARNING: Could not start ssh-agent.*')
 
+    def tearDown(self):
```

Comment:
This does not exist in cockpit-podman,machines etc. Why anaconda is special?

Reply:
My guess would be storage related testing ? That & the special environment in which Anaconda runs are usually whats results in various special cases in regards to Anaconda not only in tests.

Reply:
It is normally in testlib, but as Anaconda is already special and has it's own testing library it somehow wasn't called and I don't think we want to call Cockpit's tearDown at all.